backend/serverless.yml

```diff
@@ -67,7 +78,7 @@ functions:
           method: ANY
           private: ${self:custom.appconfig.enableApiKeyAuth}
           cors:
-            origin: "*" # <-- Specify allowed origin
+            origin: '*' # <-- Specify allowed origin
```

Comment:
ah that`s a formatting things ... might be that I didn`t use prettier. 
fine.
Will add some default settings e.g. for vscode